src/rust/ensogl/lib/components/src/list_view/entry.rs

```diff
@@ -15,15 +15,15 @@ use ensogl_theme;
 // =================
 
 /// Padding inside entry in pixels.
-pub const PADDING:f32 = 7.0;
+pub const PADDING:f32 = 14.0;
 /// The overall entry's height (including padding).
 pub const HEIGHT:f32 = 30.0;
 /// The text size of entry's labe.
 pub const LABEL_SIZE:f32 = 12.0;
 /// The size in pixels of icons inside entries.
 pub const ICON_SIZE:f32 = 0.0; // TODO[ao] restore when created some icons for searcher.
 /// The gap between icon and label.
-pub const ICON_LABEL_GAP:f32 = 7.0;
+pub const ICON_LABEL_GAP:f32 = 0.0; // TODO[ao] restore when created some icons for searcher.
```

Comment:
What does the sentence mean? "Restore" means to do something in the future. While "when created" is in the past. I don't understand it.

Reply:
I copied the comment from above. The searcher was at some point meant to contain icons. It does not at the moment, so these paddings sand gaps need to be 0 now, but they need to be set when we add icons. I've fixed the grammar, now.